### ISSUE TITLE
stripe: cc input field not always validated

### ISSUE THREAD

**Issue by ligi** [pretix/pretix#1345]
I stumbled upon the following bug in the pretix stripe plugin.
If you select the credit card radio and directly click continue the input field is not validated and you get into this screen which hangs forever (just tried long - but I project it to forever here)
![Selection_139](https://user-images.githubusercontent.com/111600/61294650-9a97f280-a7d6-11e9-89e3-af93add30b17.png)
But if you click on the credit card radio and afterwards on the inputfield it gets validated and "continue" is not working anymore (I think this is the behavior that should always be in place)
![image](https://user-images.githubusercontent.com/111600/61294792-e77bc900-a7d6-11e9-8e2c-01b6e6c3fb61.png)

**Comment by raphaelm** [comment]
@pc-coholic Can you take a look at this?

**Comment by pc-coholic** [comment]
Twofold issue here... We only enabled the payment-button once we successfully pulled the Stripe Elements Input-Field from Stripe's server.

Unfortunately, that is not when the whole form is ready to be used yet... The form is then itself polling a few more things and is only ready to be used then.

You must have somehow hit the sweetspot inbetween the loading of StripeJS and StripeJS populating all the fields of the Stripe Element.

We're now specifically listening for the Stripe Elements' `ready` signal to enable the button.

Thank you for spotting and reporting this! I bet, a few users on slow mobile network connections might have already run into this...

**Comment by ligi** [comment]
you are most welcome - thanks for the fast fix!

**Comment by jpitts** [comment]
@raphaelm and @pc-coholic, how would I install an update targeting a5ba7440fe4a7c867198a173eb5b8948843f2c08? 

My current approach to installing pretix is as follows:

```sudo su pretix
/usr/local/bin/python3 -m venv /var/pretix/venv
source /var/pretix/venv/bin/activate

(venv) pretix@ip-XXX-XX-XX-XX: pip3 install -U pip setuptools wheel
(venv) pretix@ip-XXX-XX-XX-XX: pip3 install "pretix[mysql]" gunicorn```

This is how I am trying to do it:

```(venv) pretix@ip-XXX-XX-XX-XX: pip install git+https://github.com/pretix/pretix.git@a5ba7440fe4a7c867198a173eb5b8948843f2c08```

And the error I get (the hash changes each attempt):

```Collecting git+https://github.com/pretix/pretix.git@a5ba7440fe4a7c867198a173eb5b8948843f2c08
  Cloning https://github.com/pretix/pretix.git (to revision a5ba7440fe4a7c867198a173eb5b8948843f2c08) to /tmp/pip-req-build-78ltc4_s
  Running command git clone -q https://github.com/pretix/pretix.git /tmp/pip-req-build-78ltc4_s
  Running command git submodule update --init --recursive -q
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/usr/local/lib/python3.7/tokenize.py", line 447, in open
        buffer = _builtin_open(filename, 'rb')
    FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-req-build-78ltc4_s/setup.py'
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-req-build-78ltc4_s/
```

**Comment by raphaelm** [comment]
Here's how: https://docs.pretix.eu/en/latest/admin/installation/dev_version.html

You can add the ``@a5ba7440fe4a7c867198a173eb5b8948843f2c08`` after the ``.git`` as you did.

Keep in mind that this will take you to the latest development version, which has a lot of changes over the last released version as we're currently preparing a 3.0 release. It is stable enough for production, so feel free to use it, but be prepared for a little jump :)

**Comment by jpitts** [comment]
Thanks @raphaelm !
